hbw/app/javascript/packs/hbw/components/task_list.js.jsx

```diff
@@ -1,93 +1,98 @@
+              <div className='hbw-sheet-header-title-container'>
+                <em className='hbw-sheet-header-title'>{this.t('tasks')}</em>
               </div>
-            </a>
+              <a className={a_class} href="#" onClick={this.hideWidget} title={this.t('hide_widget')}>
+                <div className="hbw-sheet-header-button-icon hbw-sheet-header-close-button-icon">
+                </div>
+              </a>
+            </div>
+          </div>
+          <div className='hbw-sheet-body'>
+            { !this.state.fetched && <Pending text={this.t('loading')} /> }
+            { this.state.tasks.length == 0 && this.state.fetched && <p>{this.t('no_tasks')}</p> }
+            { this.createGroupsChildren() }
+          </div>
+          <div className='hbw-sheet-content'>
           </div>
-        </div>
-        <div className='hbw-sheet-body'>
-          { !this.state.fetched && <Pending text={this.t('loading')} /> }
-          { this.state.tasks.length == 0 && this.state.fetched && <p>{this.t('no_tasks')}</p> }
-          { this.createGroupsChildren() }
-        </div>
-        <div className='hbw-sheet-content'>
-        </div>
-      </div>;
-    },
+        </div>;
+      },
 
-    createGroupsChildren () {
-      const { props } = this;
+      createGroupsChildren () {
+        const { props } = this;
 
-      return this.groupsFromTasks(this.state.tasks).map((group) => {
-        group.key = group.group;
-        group.env = this.props.env;
-        group.chosenTaskID = this.props.chosenTaskID;
+        return this.groupsFromTasks(this.state.tasks).map((group) => {
+          group.key = group.group;
+          group.env = this.props.env;
+          group.chosenTaskID = this.props.chosenTaskID;
 
-        return React.createElement(TaskGroup, group);
-      });
-    },
+          return React.createElement(TaskGroup, group);
+        });
+      },
 
-    groupsFromTasks (tasks) {
-      let group;
-      const groups = {};
-      for (const task of Array.from(tasks)) {
-        group = task[this.state.groupByVar];
-        if (!groups[group]) {
-          groups[group] = [];
+      groupsFromTasks (tasks) {
+        let group;
+        const groups = {};
+        for (const task of Array.from(tasks)) {
+          group = task[this.state.groupByVar];
+          if (!groups[group]) {
+            groups[group] = [];
+          }
+          groups[group].push(task);
         }
-        groups[group].push(task);
-      }
 
-      const keys = Object.keys(groups).sort((a, b) => {
-        let left;
-        let left1;
+        const keys = Object.keys(groups).sort((a, b) => {
+          let left;
+          let left1;
 
-        return ((left = a < b) != null ? left : -{ 1: ((left1 = a > b) != null ? left1 : { 1: 0 }) }); // The hell generated from CoffeeScript
-      });
+          return ((left = a < b) != null ? left : -{ 1: ((left1 = a > b) != null ? left1 : { 1: 0 }) }); // The hell generated from CoffeeScript
```

Comment:
Alarm

Reply:
I will try do something with it in #192, I promise you.